test/functional/Java16andUp/src/org/openj9/test/foreignMemoryAccess/TestCloseScope0.java

```diff
@@ -0,0 +1,164 @@
+package org.openj9.test.foreignMemoryAccess;
+
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import org.testng.Assert;
+import static org.testng.Assert.*;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.*;
+import java.lang.ref.Cleaner;
+
+import org.objectweb.asm.*;
+import static org.objectweb.asm.Opcodes.*;
+
+@Test(groups = { "level.sanity" })
+public class TestCloseScope0 {
+    public static byte[] dump() throws Exception {
+        ClassWriter cw = new ClassWriter(ClassWriter.COMPUTE_FRAMES);
+		MethodVisitor mv;
+		AnnotationVisitor av0;
+
+        cw.visit(V16, ACC_PUBLIC + ACC_SUPER, "jdk/internal/misc/RunInScoped", null, "java/lang/Object", null);
+        {
+			mv = cw.visitMethod(ACC_PUBLIC, "<init>", "()V", null, null);
+			mv.visitCode();
+			mv.visitVarInsn(ALOAD, 0);
+			mv.visitMethodInsn(INVOKESPECIAL, "java/lang/Object", "<init>", "()V");
+			mv.visitInsn(RETURN);
+            mv.visitMaxs(0, 0);
+			mv.visitEnd();
+		}
+        {
+			/**
+			 * @Scoped		// <-- package private annotation that isn't visible here.  This is why we use ASM to generate the class
+			 * public static void runInScoped(Runnable r, Scope scope) {
+			 *      r.run();
+             *      Reference.reachabilityFence(scope);
```

Comment:
indentation

Reply:
![image](https://user-images.githubusercontent.com/83662958/123482670-8ae4d380-d5d3-11eb-9059-cdda37852d75.png)
It shows up fine locally